assets/js/googlesitekit/widgets/components/WidgetAreaRenderer.js

```diff
@@ -155,16 +154,18 @@ const WidgetAreaRenderer = ( { slug } ) => {
 		}
 
 		return classNames;
-	}, [ widgets, activeWidgets ] );
+	}, [ activeWidgets ] );
+
+	if ( activeWidgets.length === 0 ) {
```

Comment:
@caseymorrisus let's move this conditional up to come after `const activeWidgets = widgets.filter( isActiveWidget );`.